cpp/include/cudf/io/writers.hpp

```diff
@@ -26,6 +26,7 @@
 #include <cudf/table/table_view.hpp>
 #include <cudf/types.hpp>
 #include <io/parquet/parquet.h>
+#include <io/orc/orc.h>
 
```

Comment:
Yup, ended up including writer_impl.hpp from functions.cpp rather than writer types from orc.h, though I'm thinking maybe the chunked_state definition should go into its own header to limit unnecessary dependencies (maybe "io/{parquet|orc}/chunked_state.h" ?).

Reply:
Ended up moving orc_chunked_state in its own orc/chunked_state.hpp header